.pre-commit-config.yaml

```diff
@@ -2,14 +2,11 @@ repos:
   - repo: https://github.com/gruntwork-io/pre-commit
     # Stick to v0.1.20 until this bug is fixed: https://github.com/gruntwork-io/pre-commit/issues/102
```

Comment:
i guess this comment is not valid anymore right?
```suggestion
```

Reply:
yup - it is not needed anymore

Reply:
so let's remove it, you can commit my suggestion